content/ru/_index.html

```diff
@@ -41,12 +41,12 @@ <h2>О сложности миграции 150+ микросервисов в Ku
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Смотреть видео</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Посетите KubeCon в Северной Америке, 11-15 октября 2021 года</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2022/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton">Посетите KubeCon в Европе, 17-20 мая 2022 года</a>
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2022/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton" button id="desktopKCButton">Посетите KubeCon в Европе, 17-20 мая 2022 года</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton" button id="desktopKCButton">Посетите KubeCon в Северной Америке, 24-28 октября 2022 года</a>
```

Comment:
`kccncna21` doesn't look right. Is it wrong in upstream (English)?

Reply:
I think it _is_ wrong in upstream. This is therefore correct in terms of localization.
/lgtm

Reply:
ok